docs/CONTRIBUTING.md

```diff
@@ -35,6 +35,10 @@ is a great starting point if you are new to version control.
    - `upstream`, which refers to the `insarlab` repository
    - `origin`, which refers to your personal fork
 
++ Setting up `pre-commit` within `MintPy` directory:
```

Comment:
I add `pre-commit sample-config` here to generate an initial basic config file before `pre-commit install`.
This is suggested in https://pre-commit.com/#2-add-a-pre-commit-configuration

Otherwise, we will have this message when doing `git commit`:
```
No .pre-commit-config.yaml file was found
- To temporarily silence this, run `PRE_COMMIT_ALLOW_NO_CONFIG=1 git ...`
- To permanently silence this, install pre-commit with the --allow-missing-config option
- To uninstall pre-commit run `pre-commit uninstall`
```